### PR TITLE
add task 14 sum oll number in string

### PR DESCRIPTION
**Pull request by Ksenia000**

### CODE REVIEW

File: Learn/js/computeSumInString_script.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+function SumInString() {
```
Comment (akryvenko) [inline Learn/js/computeSumInString_script.js]: rename to sumNumversInString

File: Learn/js/computeSumInString_script.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    var str1 = document.getElementById("str1").value;
+    var arrStr = Array.from(str1);
+    var res = 0;
+    var lgth = arrStr.length;
+    for (var i = 0; i < lgth; i++) {
+        if (Check(arrStr[i]) == true) {
```
Comment (akryvenko) [inline Learn/js/computeSumInString_script.js]: please use the camelcase for function and variable names. For more details please read this https://www.w3schools.com/js/js_conventions.asp

**Review by akryvenko — CHANGES_REQUESTED** [review]
Please take into account the following scenario 10dd22ff50g-1 result should be 81.

File: Learn/js/computeSumInString_script.js (hunk 1 above)
Comment (Ksenia000) [reply Learn/js/computeSumInString_script.js]: ))

in the beginning I named it, but then I thought that it was a very long name and renamed.
ok will fix the code

Comment (akryvenko) [reply Learn/js/computeSumInString_script.js]: long names it is good practice, short not always, try to explain what function do in function do, e.g. extractAndSumFumbersInString

**Pull request merged**